miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/view/MiniAppViewImpl.kt

```diff
@@ -65,4 +66,17 @@ internal class MiniAppViewImpl(
             }
         }
     }
+
+    override fun sendJsonToMiniApp(message: String, onFailed: () -> Unit) {
+        message.let {
+            if (it.isNotBlank()) {
+                miniAppParameters.config.miniAppMessageBridge.dispatchNativeEvent(
+                    NativeEventType.MINIAPP_RECEIVE_JSON_INFO,
+                    message
+                )
```

Comment:
```suggestion
when (miniAppParameters) {
                    is MiniAppParameters.DefaultParams -> (miniAppParameters as MiniAppParameters.DefaultParams).config.miniAppMessageBridge.dispatchNativeEvent(
                        NativeEventType.MINIAPP_RECEIVE_JSON_INFO,
                        message
                    )
                    is MiniAppParameters.InfoParams -> (miniAppParameters as MiniAppParameters.InfoParams).config.miniAppMessageBridge.dispatchNativeEvent(
                        NativeEventType.MINIAPP_RECEIVE_JSON_INFO,
                        message
                    )
                    is MiniAppParameters.UrlParams -> (miniAppParameters as MiniAppParameters.UrlParams).config.miniAppMessageBridge.dispatchNativeEvent(
                        NativeEventType.MINIAPP_RECEIVE_JSON_INFO,
                        message
                    )
                }
```

can you change it to these,  adding overridden parameter in the `sealed` class is not necessary now, i am a bit concerned that `config` might be effected the parameters inside it, but we can improve this later with some more tests. thank you.